docs/backend/testing/running-integration-tests-with-routing-middleware-enabled.md

```diff
@@ -0,0 +1,105 @@
+---
+sidebar_position: 8
+---
+
+# Running integration tests with routing middleware enabled
+
+Sometimes it will be necessary to write server integration tests with routing middleware enabled. For example, we want to test calling an API endpoint with an unauthorized user, and ensure that we receive a 403 response. Here's how to go about it.
```

Comment:
**suggestion (non-blocking)**:

```suggestion
Sometimes it will be necessary to write server integration tests with routing middleware enabled. For example, when we want to test calling an API endpoint with an unauthorized user, and ensure that we receive a 403 response. Here's how to go about it.
```

Reply:
Good suggestion, I updated it to "we may want."